contrib/dist/debian/liquidprompt/DEBIAN/control

```diff
@@ -1,11 +1,11 @@
 Package: liquidprompt
-Version: 1.3-0
+Version: 2.2.0
 Section: base
 Priority: optional
 Architecture: all
 Depends: bash
 Recommends: acpi
-Suggests: git, screen, tmux
+Suggests: git, svn, fossil, mercurial, bzr, cmake, screen, tmux, kubernetes
```

Comment:
`kubernetes` is not a package usually, or if it is, it's the server.

What we depend on is `kubectl`, which is normally not installed via package management. But if it was, it would be called `kubectl`: https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-using-native-package-management